app/controllers/sessions_controller.rb

```diff
@@ -12,6 +12,7 @@ def callback
         provider: Provider.create_or_find_by(
           name: "DfE",
           dttp_id: "00000000-0000-0000-0000-000000000000",
+          code: "000",
```

Comment:
🤮 This stuff is 'orrible. Let's leave it for now but I think we should consider removing this feature as it's just going to create broken users/providers.

Reply:
@gpeng took me hours to figure out why one of the tests was failing